server-common/src/main/java/org/apache/kafka/server/share/persister/PersisterStateManager.java

```diff
@@ -467,7 +471,7 @@ protected void handleFindCoordinatorResponse(ClientResponse response) {
 
                 default:
                     log.error("Unable to find coordinator for {} using key {}.", name(), partitionKey());
-                    findCoordinatorErrorResponse(error, null);
+                    findCoordinatorErrorResponse(error, new Exception(errMessage));
```

Comment:
Why did you choose to pass the error message for this case but not the other RPCs? I would have expected consistent failing in all cases.

Reply:
Will update